client/src/app/+videos/+video-watch/video-watch.component.ts

```diff
@@ -799,6 +799,10 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
       common: {
         autoplay: this.isAutoplay(),
         nextVideo: () => this.zone.run(() => this.autoplayNext()),
+        previousVideo: () => this.zone.run(() => {
```

Comment:
Assign this property in a dedicated `if` below `const option = {` so you don't display it when it's not a playlist